test/e2e/network/service.go

```diff
@@ -3799,18 +3799,10 @@ var _ = SIGDescribe("SCTP [Feature:SCTP] [LinuxOnly]", func() {
 	})
 
 	ginkgo.It("should create a Pod with SCTP HostPort", func() {
-		ginkgo.By("checking whether kubenet is used")
 		node, err := e2enode.GetRandomReadySchedulableNode(cs)
 		framework.ExpectNoError(err)
 		hostExec := utils.NewHostExec(f)
 		defer hostExec.Cleanup()
-		cmd := "ps -C kubelet -o cmd= | grep kubenet"
-		framework.Logf("Executing cmd %q on node %v", cmd, node.Name)
-		err = hostExec.IssueCommand(cmd, node)
-		if err != nil {
-			e2eskipper.Skipf("Interrogation of kubenet usage failed on node %s", node.Name)
-		}
-		framework.Logf("kubenet is in use")
```

Comment:
I'd like to know why this original check was implemented before. Is this just meaningless?
From https://kubernetes.io/docs/concepts/extend-kubernetes/compute-storage-net/network-plugins/#support-hostport CNI plugins also support HostPort and this test can be independent from kubenet, is that correct?

/cc @oomichi

Reply:
you have more history here https://github.com/kubernetes/kubernetes/issues/92041#issuecomment-644100485

It seems that not all runtimes were allowing SCTP

Reply:
Then do we need to add check of runtimes instead of kubenet?

Reply:
since SCTP is a GA feature and per https://github.com/kubernetes/kubernetes/issues/92041#issuecomment-644092139 the runtimes using portmap plugin shoud implement it (most of them), I would not add a skip and 
defer  people to use tags to filter the test if it fails. They can remove the tag filter once they fix/implement the feature, if we add a skip detector ... this is an example, I found this skip by chance :smile: 
I have a SCTP job with containerd and it' s working,  upstream CI will not fail.

Reply:
it's behind a `[Feature:SCTP]` tag and people commonly need to skip Feature already.